### PR TITLE
✨ (Feature): Static Au Pair screen and basic modal

### PR DESCRIPTION
**Pull request by Ruben-Brits**

### CODE REVIEW

File: apps/the-au-pair/src/styles.scss [hunk 1]
```diff
@@ -50,3 +50,8 @@ ion-button{
     line-height: 23px !important;
     --color: white !important;
 }
+
```
Comment (ricecakesreece) [inline apps/the-au-pair/src/styles.scss:53]: Unnecessary spaces

File: libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      </ion-card> 
+    </div>
+  </div>
+
+  <div class="mobile-container">
+    <ion-slides id="schedule-grid-mobile" [options]="{ slidesPerView: 'auto', zoom: false,loop: true, spaceBetween: 16 }">
+    <ion-slide *ngFor="let day of [0,1,2,3,4,5,6]" style="width: 90%; height: 81vh;">
+      <ion-col>
+        <div class="day ion-text-center">
+          <h2 class="heading"> {{ days[day] }}</h2>
```
Comment (ricecakesreece) [inline libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.html:30]: Just keep consistent with spacing as you did with {{item.name}} above

File: libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+              </div>
+            </div>
+          </ion-card> 
+        </div>
+      </ion-col>
+    </ion-slide>
+  </ion-slides>
+  </div>
+  
+
```
Comment (ricecakesreece) [inline libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.html:50]: Unnecessary spaces

File: libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.scss [hunk 4]
```diff
@@ -0,0 +1,121 @@
+.text-white{
```
Comment (ricecakesreece) [inline libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.scss:1]: indent

File: libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.scss [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    box-shadow: none;
+  }
+  
+  #schedule-grid-desktop{
+    display: grid;
+    gap: 2px;
+    grid-template-columns: repeat(7,minmax(250px,1fr));
+    width: 95%;
+    margin-inline: auto;
+
```
Comment (ricecakesreece) [inline libs/au-pair-schedule/feature/src/lib/au-pair-schedule.component.scss:50]: Unnecessary space

File: apps/the-au-pair/src/styles.scss [hunk 6]
```diff
@@ -50,3 +50,8 @@ ion-button{
...
+
+.ion-page{
+    background-clip: content-box;
+}
```
Comment (PxttytheThxtty) [inline apps/the-au-pair/src/styles.scss:57]: No newline at end of file

**Review by ricecakesreece — APPROVED** [review]
Very nice

**Pull request merged**